projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-controls/index.tsx

```diff
@@ -96,44 +95,132 @@ type AiAssistantControlComponentProps = {
 	key?: AiAssistantKeyProp | string;
 
 	/*
-	 * The label to use for the dropdown. Optional.
+	 * The block type. Required.
 	 */
-	label?: string;
+	blockType: ExtendedBlockProp;
 
 	/*
-	 * A list of quick edits to exclude from the dropdown.
-	 */
-	exclude?: AiAssistantKeyProp[];
-
-	/*
-	 * Whether the dropdown is requesting suggestions from AI.
+	 * The label to use for the dropdown. Optional.
 	 */
-	requestingState?: string;
+	label?: string;
 
 	/*
 	 * Whether the dropdown is disabled.
 	 */
 	disabled?: boolean;
```

Comment:
label, key, and disabled can all be removed I think?

Reply:
yes, we can work on the behavior of the toolbar button disable here